src/main/java/com/internet/shop/model/Role.java

```diff
@@ -0,0 +1,31 @@
+package com.internet.shop.model;
+
+public class Role {
+    private long id;
+    private RoleName roleName;
+
+    public Role(RoleName roleName) {
```

Comment:
```suggestion
    private Role(RoleName roleName) {
```

let's create static `of()` method that will use this constructor